### PR TITLE
Light Mode Interaction States - NxTextInput - RSC-1482

### PR DESCRIPTION
**Pull request by avnvbhatta**
https://issues.sonatype.org/browse/RSC-1482

### CODE REVIEW

File: gallery/visualtests/nxTextInput.js [hunk 1]
```diff
@@ -32,10 +32,11 @@ describe('NxTextInput', function() {
 
     it('has a darker border when hovered',
         hoverTest(simpleComponentSelector, getInputElementSelector(simpleComponentSelector)));
-    it('has a blue border and glow when focused',
+
+    it('has a blue inner outline when focused',
         focusTest(simpleComponentSelector, getInputElementSelector(simpleComponentSelector)));
 
-    it('has a blue border and glow when hovered and focused',
+    it('has a blue inner outline when hovered and focused',
```
Comment (EmilyBermudez88) [inline gallery/visualtests/nxTextInput.js:39]:
```suggestion
    it('has a darker border and blue inner outline when hovered and focused',
```

**Comment by EmilyBermudez88** [comment]
I'm wondering if the snapshots & tests for all the other components should be left out of this ticket and dealt with in their respective tickets?

**Comment by avnvbhatta** [comment]
> I'm wondering if the snapshots & tests for all the other components should be left out of this ticket and dealt with in their respective tickets? Or at least edit the ticket in Jira to specify which components are being dealt with here.

I didn't know that the other components had their respective tickets. Reverted here: 24b8de12a339c45fa58fc954b2d1da7c8f506ac6

**Review by EmilyBermudez88 — APPROVED** [review]
LGTM!

**Pull request merged**